code/game/objects/items/stacks/medical.dm

```diff
@@ -95,37 +106,44 @@
 		M.heal_bodypart_damage((heal_brute/2))
 		return TRUE
 	if(iscarbon(M))
-		return heal_carbon(M, user, heal_brute, 0)
-	to_chat(user, "<span class='warning'>You can't heal [M] with the \the [src]!</span>")
+		return heal_carbon(M, user, heal_brute, heal_burn)
+	to_chat(user, "<span class='warning'>You can't heal [M] with \the [src]!</span>")
 
 /obj/item/stack/medical/bruise_pack/suicide_act(mob/user)
 	user.visible_message("<span class='suicide'>[user] is bludgeoning [user.p_them()]self with [src]! It looks like [user.p_theyre()] trying to commit suicide!</span>")
 	return (BRUTELOSS)
 
 /obj/item/stack/medical/gauze
 	name = "medical gauze"
-	desc = "A roll of elastic cloth that is extremely effective at stopping bleeding, but does not heal wounds."
+	desc = "A roll of elastic cloth, perfect for stabilizing all kinds of wounds, from cuts and burns, to broken bones. "
 	gender = PLURAL
 	singular_name = "medical gauze"
 	icon_state = "gauze"
-	var/stop_bleeding = 1800
-	self_delay = 20
-	max_amount = 12
+	self_delay = 50
+	other_delay = 20
+	max_amount = 8
+	amount = 4
 	grind_results = list(/datum/reagent/cellulose = 2)
 	custom_price = 100
+	absorption_rate = 0.25
+	absorption_capacity = 5
+	splint_factor = 0.25
+
+// gauze is only relevant for wounds, which are handled in the wounds themselves
+/obj/item/stack/medical/gauze/try_heal(mob/living/M, mob/user, silent)

```

Comment:
just so you know, people were using gauze before to stop corpses from bleeding while being dragged around on the ground, and this basically removes that usage from the game

sure, you can just use body bags, but gauze is easier to apply to a moving target (such as one that's actively being dragged across the station by a dumbass who doesn't know what he's doing)